src/modules/error/components/Failure.js

```diff
@@ -19,19 +18,17 @@ margin-bottom: 10px;
 `
 
 type PropsType = {|
-  error: Error,
+  error?: Error,
   t: TFunction
 |}
 
 export class Failure extends React.Component<PropsType> {
   render () {
-    const { t } = this.props
+    const { t, error } = this.props
 
     return <ViewContainer>
       <IconContainer name='sentiment-dissatisfied' size={55} color={'black'} />
```

Comment:
Replace that icon with this one: https://drive.google.com/drive/u/1/folders/1CUb_SWII1WhLzbm2_z99WtCqTXpdRHrO

I think it makes sense to do it in this task.